testing/app.go

```diff
@@ -50,10 +50,10 @@ type TestingApp interface {
 	LastBlockHeight() int64
 }
 
-func SetupTestingApp() (TestingApp, map[string]json.RawMessage) {
+func SetupTestingApp(chainID string) (TestingApp, map[string]json.RawMessage) {
```

Comment:
I think we should try to avoid breaking API here, it also requires anyone replacing the `DefaultTestingAppInit` to replicate these changes. Would it be possible to set the chainID for baseapp in `NewTestChain`, [here](https://github.com/cosmos/ibc-go/blob/carlos/bump-sdk-v0.47/testing/chain.go#L180). After the test chain is returned, I believe you can do `tc.App.GetBaseApp().SetChainID(chainID)`

Reply:
see #3297